runelite-client/src/main/resources/net/runelite/client/plugins/gpu/frag.glsl

```diff
@@ -58,4 +60,7 @@ void main() {
   } else {
     FragColor = smoothColor;
   }
+
+  vec3 mixedColor = mix(FragColor.rgb, fogColor.rgb, fogAmount);
```

Comment:
I would avoid trying to reading from 'out' variables since I'm not sure how well that is supported.

Reply:
Yeah, fixed